web/src/app/main/components/ToolbarTitle.js

```diff
@@ -94,9 +95,10 @@ NameLoader.propTypes = {
 function ToolbarTitle() {
   const width = useWidth()
   const classes = useStyles()
+  const [applicationName] = useConfigValue('General.ApplicationName')
```

Comment:
Fall back to the value from `index.html` if empty; maybe add it as a global var like public path.